app/models/tag.rb

```diff
@@ -170,7 +170,11 @@ def self.followed_tags_for(follower:)
       .order(hotness_score: :desc)
   end
 
-  def self.followed_by(user, points = (1..))
```

Comment:
I just want to bring to attention that we were using the following methods:
 followed_by(user, points = (1..)) which was returning tags as followed if the points  >=1 and
 self.antifollowed_by(user) which was returning tags as antifollowed if the points are <1 .
The problem with this is that we are currently allowing explicit_points/weights to be set between 0 to 1 on the /dashboard/following_tags page but in places where we’re using these methods directly or indirectly like in cached_followed_tag_names and cached_antifollowed_tag_names we were omitting tags that had points that lay between these 1 and -1.
With the updated requirements as per [Hide tag weights and change negative weight tags to “Hidden tags” #19675](https://github.com/forem/forem/issues/19675) we’ve changed this be (0…) and (…0) which is greater than and including 0 for followed tags and less than 0 for antifollowed/hidden tags.

I want to note this as it may change the results in places where these methods are being used. I dropped this in #product-management as well.

Reply:
Thanks for this anticipation!